contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+.. note::
+
+   SWAP excels at its simplicity. It is not the goal and would completely defeat the purpose of the system to introduce bullet-proof consistency and auditing - Swarm is not a blockchain in its own terms.`
+
+
+.. important::
+   The Swarm node operator is responsible for maintenance of the local accounting database (a LevelDB instance). Tampering with it, altering its contents or removing it altogether may result in the node becoming inoperable.
+
+If a node requests data from a peer, when the peer delivers the data, the peer credits an amount equivalent to the price of that message in its local database, at the index representing the node it delivers to. The node, when it receives the message (data) from the peer, debits itself in its local database the same amount at the index of the peer. 
+
+.. image:: img/swap.svg
+   :alt: Simple SWAP exchange between two nodes 
+   :width: 400
+
+In the above example, A requested data from B, which B then delivers to A (could be a chunk). Assuming a price of 200 units for the chunk, B credits itself in its local database plus 200 at the index for node A, while A, when it finally receives the data from B, debits minus 200 in its local database at the index for node B.
+
+Due to the distributed nature of Swarm nodes, entries in the local databases do not happen simultaneously, and there is no notion of transactions nor confirmations; in normal operation, both nodes account for the message with the correspondent node at the moment of the action (for the sending peer, at the moment of sending, for the receiving peer, at the moment of receiving). 
+
+If a message is being sent, if we would account for it before sending, then a failure in sending would require a complex rollback (or an imbalance). If we send first and then account, then it would not be possible to actually check if there are enough funds. Receiving a message incurs in the same challenge. To address this, Swarm first does a read operation on the database to check if there are enough funds. If positive, the requested operation (send/receive) is performed, and finally if the operation succeeded, the accounting entry is persisted.
+
+Thresholds
+----------
+As described in the [`Incentives system for Swarm <https://swarm-gateways.net/bzz:/swarm.eth/ethersphere/orange-papers/1/sw%5E3.pdf>`_], this accounting system works by allowing the mutual accounting to oscillate in a defined range. This range is delimited by two thresholds:
+* The payment threshold is a number at which if the node goes below that relative to a peer, it should trigger a payment to that peer (a cheque, see below). This is initiated by the debitor node.
+* The disconnect threshold is a number at which if the node goes above that number relative to a peer, it can (it's up to the peer but this is the expected behavior) disconnect the peer. This is monitored by the creditor.
+
+The disconnect threshold amount should be bigger than the payment threshold in absolute terms.
```

Comment:
I would say "must" here instead of "should". We even check for this during startup, the node won't even start if the defined `PaymentThreshold` is higher than the `DisconnectThreshold`.